lib/src/utils/marketplace_token.ts

```diff
@@ -64,6 +64,27 @@ export const isTokenValid = () => {
     }
     return false
   } catch (error) {
+    Logger.error(error)
     return false
   }
 }
+
+/**
+ * Return the address that sign the token
+ * @return The address token signer 
+ */
+export const getAddressTokenSigner = () => {
+  try {
+    const { token } = fetchMarketplaceApiTokenFromLocalStorage()
+    if (token && jwt.decode(token)) {
```

Comment:
```suggestion
    if (token) {
```
I don't think this is needed